flake.nix

```diff
@@ -0,0 +1,88 @@
+# To develop:
+#
+# nix develop
+# cmake -B build -GNinja
+# ninja -C build
+#
+# To build:
+# nix build .#static
+# nix build .#shared
+
+{
+  inputs = {
+    # we need this to access nix packages
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    # we need this to build for each system
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        name = "liboqs";
+        version = "0.8.0";
```

Comment:
Why 0.8.0?

Reply:
Good point! I'm not sure I actually recall why I chose that. Do you have a good recommendation for the version?

Reply:
Currently this is "0.11.0" -- but it seems weird to have to hard-code this (and then have to manually change going forward): Any chance to automatically take this from the `cmake` logic?

Reply:
"Resolved" as version tag is gone.